recipes/logr/v0.7/conanfile.py

```diff
@@ -32,7 +32,7 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("fmt/10.1.1")
+        self.requires("fmt/[>=9.1]")
```

Comment:
```suggestion
        self.requires("fmt/10.1.1")
```

Please, revert. Version ranges is allowed only to few packages: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges


About your conflict version error, we will need to open a second PR to update other dependencies.